jmix-dynattr/dynattr-flowui/src/main/java/io/jmix/dynattrflowui/view/categoryattr/CategoryAttributesDetailView.java

```diff
@@ -368,6 +406,8 @@ protected void onEditEnumerationBtnClick(ClickEvent<Button> event) {
                         AttributeEnumerationDetailView screen = afterCloseEvent.getSource().getView();
                         getEditedEntity().setEnumeration(screen.getEnumeration());
                         getEditedEntity().setEnumerationLocales(screen.getEnumerationLocales());
+                        defaultEnumField.getDataProvider().refreshAll();
+                        defaultEnumField.clear();
```

Comment:
Why do you clear the default value after you added some possible enum value? For example my enum was A, B, C. The default value was A. Why after adding the D option the default A is erased?